apps/camera/index.html

```diff
@@ -29,6 +29,7 @@
     <!-- <script src="/shared/js/media/media_frame.js"></script> -->
     <!-- <script src="/shared/js/media/downsample.js"></script> -->
     <!-- <script src="/shared/js/media/image_size.js"></script> -->
+    <!-- <script src="/shared/js/media/jpeg-exif.js"></script> -->
```

Comment:
I don't see any code that modifies the camera pick activity to use image/jpeg+exif. I think that means that this file will never be lazy loaded in the camera app.  So I suspect you can delete this line. 

So I'd suggest that you either:

1) delete this line and file a followup bug for the camera pick activity.

2) modify the camera pick activity to use image/jpeg+exif. I think that change would go in apps/camera/js/lib/resize-image-and-save.js